src/shared/views/ChangeDetection/InstructionsScreen.js

```diff
@@ -118,6 +121,13 @@ class CDInstructionScreen extends React.Component<Props> {
                 </View>
 
                 <ScrollView style={styles.container}>
+                    <Text style={styles.header}>
+                        <Trans i18nKey="CDInstructionScreen:imageryCredits">
```

Comment:
Any reason why you use `Trans` here instead of the `t()` function? The text is simple enough for the latter, I think.

Reply:
I'm not sure if there is any difference between the two. For me trans was sometimes a bit more obvious as you can place default values. That helped during the initial implementation, but now I could also use only t().

Or is there any other difference between the two?